dbmgr/dbmgr.go

```diff
@@ -341,6 +341,7 @@ type Content struct {
 	UserID      UserID
 	Description string
 	Size        int64
+	Dir         bool
```

Comment:
We probably want to have this be more than a bool, Users can (and do) pin things outside of files and directories, like blockchain segments, random ipld graphs, cbor maps of things, and so on.
Maybe for now we can make it be an integer field called `Kind` ('Type' overloaded and overused). It can be any of (for now) unknown=0, file=1, directory=2 and we can add more types later if needed

Reply:
+1 to this, @gmelodie maybe similar to your proposal in this PR:
https://github.com/application-research/estuary/pull/100/files#diff-56307bdb7ded67ebe516f562b90384407b21b2d34e8b5766b42602e1fb9705f0R4161

Reply:
Agreed! I personally think `Type` is a good name since it's inside the `Content` struct all of the time, otherwise I used `contentType` when handling it outside of these structs. Here's what I did:
1. Added a `Type` field to `Content` and set three types (`Unknown`, `File` and `Directory`)
2. Changed the logic to use the new `Type` field and still keep the response of `handleColsFsDir` intact, returning a bool field `dir`